homeassistant/components/vallox/__init__.py

```diff
@@ -291,3 +316,26 @@ async def async_handle(self, call: ServiceCall) -> None:
         # be observed by all parties involved.
         if result:
             await self._coordinator.async_request_refresh()
+
+
+class ValloxEntity(CoordinatorEntity[ValloxDataUpdateCoordinator]):
+    """Representation of a Vallox entity."""
+
+    def __init__(self, name: str, coordinator: ValloxDataUpdateCoordinator) -> None:
+        """Initialize a Vallox entity."""
+        super().__init__(coordinator)
+
+        self._device_uuid = self.coordinator.data.get_uuid()
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, str(self._device_uuid))},
+            manufacturer=DEFAULT_NAME,
+            model=self.coordinator.data.get_model(),
+            name=name,
+            sw_version=self.coordinator.data.get_sw_version(),
+        )
+
+        if self.coordinator.config_entry is not None:
+            self._attr_device_info[
+                "configuration_url"
```

Comment:
Please use the constant

`homeassistant/const.py:ATTR_CONFIGURATION_URL: Final = "configuration_url"`

Reply:
Done